src/test/denialofservice_tests.cpp

```diff
@@ -356,7 +353,7 @@ BOOST_AUTO_TEST_CASE(peer_discouragement)
                          ConnectionType::INBOUND,
                          /*inbound_onion=*/false};
     nodes[0]->SetCommonVersion(PROTOCOL_VERSION);
-    peerLogic->InitializeNode(nodes[0], NODE_NETWORK);
+    peerLogic->InitializeNode(nodes[0], ServiceFlags(NODE_NETWORK | NODE_WITNESS));
```

Comment:
d4759a02b51368870526f04558797bb7f5749c73: Looks like you are changing the value, why?

Reply:
All the service flags should now have the same values as before.